autoscale-docker-swarm-container/scaler.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+logging:
+  level: INFO
+  loggers:
+    com.hpe.caf: DEBUG
```

Comment:
default was taken from the marathon container in the same repo, I was keeping the logging consistent.  Should it also be changed?  https://github.com/Autoscaler/autoscaler/blob/develop/autoscale-container/scaler.yaml